src/api/evaluate/evaluate.py

```diff
@@ -6,7 +6,7 @@
 from pathlib import Path
 from datetime import datetime
 from promptflow.core import AzureOpenAIModelConfiguration
```

Comment:
We should remove this, I don't think `AzureOpenAIModelConfiguration` is being used.